chore(blog): docs June report (#1908)
asyncapi/website: pages/blog/2023-june-docs-report.md

```mermaid
graph LR
    A(("AsyncAPI Docs' Hall of Fame"))
    B["Inaugural Member"]
    C["Florence Njeri"]
    D["Role: Core Generator Tools Docs Maintainer"]
    E["Key Contribution: Creation & Maintenance of Generator Tool Docs"]
    F["Active Member of AsyncAPI TSC"]
    G["Impecable Responsiveness as Core Docs Maintainer"]
    H["Graduate of 2022 Google Season of Docs (AsyncAPI)"]
    I["Created Most Documentation in 2022 GSoD"]
    J["Automated Generator Docs Copying via asyncapi/generator#872"]
    A --> B
    B --> C
    C --> D
    C --> E
    C --> F
    C --> G
    C --> H
    H --> I
    H --> J
    style A fill:#C1F9F7,stroke:#333,stroke-width:4px 
    style B fill:#80c6ff,stroke:#333,stroke-width:2px
    style C fill:#E1FEC1,stroke:#333,stroke-width:2px
    style D fill:#FF6EC7,stroke:#333,stroke-width:2px
    style E fill:#FF6EC7,stroke:#333,stroke-width:2px
    style F fill:#FF6EC7,stroke:#333,stroke-width:2px
    style G fill:#FF6EC7,stroke:#333,stroke-width:2px
    style H fill:#FF6EC7,stroke:#333,stroke-width:2px
    style I fill:#DAD0FF,stroke:#333,stroke-width:2px
    style J fill:#DAD0FF,stroke:#333,stroke-width:2px
```

## AsyncAPI Docs - June Summary

> June 2023 was a pivotal month for AsyncAPI's vibrant documentation community. The month marked the introduction of our **AsyncAPI Docs' Hall of Fame**, with [Florence Njeri](https://github.com/Florence-Njeri) being the inaugural inductee for her exceptional contribution. We've also unveiled the **'Technical Writing'** category as part of our 2023 Mentorship Program, welcoming Cynthia Peter, Arya Gupta, Joy Almeida, and Vaishnavi Nandakumar.

Author: ```suggestion
June 2023 was a pivotal month for AsyncAPI's vibrant documentation community. The month marked the introduction of our **AsyncAPI Docs' Hall of Fame**, with [Florence Njeri](https://github.com/Florence-Njeri) being the inaugural inductee for her exceptional contribution. We've also unveiled the **`Technical Writing`** category as part of our 2023 Mentorship Program, welcoming Cynthia Peter, Arya Gupta, Joy Almeida, and Vaishnavi Nandakumar.
```

---
title: "AsyncAPI Docs Report - June 2023"
date: 2023-07-10T06:00:00+01:00
type: Engineering
tags:
  - Documentation
  - Communication
cover: /img/posts/2023-june-docs-report.webp
authors:
  - name: Alejandra Quetzalli
    photo: /img/avatars/canela-ale.webp
    link: https://www.linkedin.com/in/alejandra-quetzalli/
    byline: In June 2023, the AsyncAPI documentation had 7,827 sessions and 4,203 unique users, including 2,566 new users.
featured: true
---

# AsyncAPI Docs Report - June 2023
In June 2023, the AsyncAPI documentation had **7,827 sessions** and **4,203 unique users**, including **2,566 new users**, showing consistent engagement within our vibrant docs community. 

> Adding to this excitement was the kick-off of both the _2023 Google Season of Docs_ and _Technical Writing Mentorship Program_ projects, further expanding our bustling docs community.

Reviewer: ```suggestion
Adding to this excitement was the kick-off to both the _2023 Google Season of Docs_ and _Technical Writing Mentorship Program_ projects, further expanding our bustling docs community.
```
- How's this? @alequetzalli

Author: The original is grammatically correct 😄

Reviewer: Okay then

## Google Season of Docs 2023 Update: AsyncAPI Projects

> As of June, our Google Season of Docs (GSoD) writers have been diligently working on tasks from the [cover AsyncAPI document sections in detail](https://github.com/asyncapi/website/issues/1507) project. Most have completed initial drafts for their first documents and some are already diving into their second. A select few have gone above and beyond, arranging discussions with core spec contributors to clarify questions on their docs pull requests.

Reviewer 2: ```suggestion
As of June, our Google Season of Docs (GSoD) writers have been diligently working on tasks from the [cover AsyncAPI document sections in detail](https://github.com/asyncapi/website/issues/1507) project. Most have completed initial drafts for their first documents, and some are already diving into their second. A select few have gone above and beyond, arranging discussions with core spec contributors to clarify questions on their docs pull requests.
```

> Meanwhile, progress on the [create 100-level interactive learning path](https://github.com/asyncapi/website/issues/1520) project continues. We've concluded the [community vote on our host platform for AsyncAPI Interactive Learning Paths](https://github.com/orgs/asyncapi/discussions/711), with Killercoda securing the majority of votes. Consequently, our writers are now fully geared towards crafting the 2023 GSoD Learning Path on the Killercoda platform. Once developed, we can repurpose this content to also share on the Linux Foundation platform.

Reviewer: ```suggestion
Meanwhile, progress on the [create 100-level interactive learning path](https://github.com/asyncapi/website/issues/1520) project continues. We've concluded the [community vote on our host platform for AsyncAPI Interactive Learning Paths](https://github.com/orgs/asyncapi/discussions/711), with [Killercoda](https://killercoda.com) securing the majority of votes. Consequently, our writers are now fully geared towards crafting the 2023 GSoD Learning Path on the Killercoda platform. Once developed, we can repurpose this content to also share on the Linux Foundation platform.
```
- Can we add the website link for Killercoda?

> > **Note:** For clarity, the 2023 GSoD focus lies solely on implementation on one platform—Killercoda. Any potential addition of the content to the Linux Foundation platform is a separate task, not included in the 2023 GSoD writers' obligations.

Reviewer: ```suggestion
> **Note:** For clarity, the 2023 GSoD focus lies solely on implementation on one platform—Killercoda. Any potential addition of the content to the Linux Foundation platform is a separate task and is not included in the 2023 GSoD writers' obligations.
```

Author: Don't like this change

Reviewer: Okay, got it.

## Introducing a `Technical Writing` Category in 2023 AsyncAPI Mentorship Program
In our quest to continually enhance the AsyncAPI Mentorship Program, we're excited to announce the inclusion of a `Technical Writing` category in our 2023 edition. For 2023, the selected technical writers will document how various protocols _(i.e., Kafka, Websocket, MQTT, etc.)_ work with the AsyncAPI specification.

We're delighted to welcome and congratulate our accepted writers into this year's program - [Cynthia Peter](https://github.com/CynthiaPeter), [Arya Gupta](https://github.com/Arya-Gupta), [Joy Almeida](https://github.com/J0SAL), and [Vaishnavi Nandakumar](https://github.com/VaishnaviNandakumar). Their remarkable skills and passion shone through during a rigorous selection process, earning them their well-deserved places in our 2023 Technical Writing Mentorship Program.

## Join our `OPEN` Spec 3.0 Docs Meetings
Calling all AsyncAPI community members! We're continuing our series of **Spec 3.0 documentation** community meetings, happening **biweekly on Thursdays at 14:30 UTC**.

> Whether you're eager to contribute to Spec 3.0 docs, or you have questions that need answers, these open meetings are your gateway to connect and collaborate with our expert spec contributors. We especially encourage our GSoD and Mentorship Program writers to attend to ensure their contributions align flawlessly with Spec 3.0.

Reviewer: ```suggestion
Whether you're eager to contribute to Spec 3.0 docs, or you have questions that need answers, these open meetings are your gateway to connect and collaborate with our expert spec contributors. We especially encourage our GSoD and Mentorship Program writers to attend so as to ensure that their contributions align flawlessly with Spec 3.0.
```

A hearty congratulations to [Florence Njeri](https://github.com/Florence-Njeri), our inaugural inductee and exceptional docs contributor for June. Florence, an active TSC member, excels in her meticulous maintenance of the [Generator Tool documentation](https://www.asyncapi.com/docs/tools/generator). Her consistent, flawless, and timely contributions have made her an invaluable asset to our community.

Florence, a Kenyan 🇰🇪 engineer, emerged from the 2022 Google Season of Docs with AsyncAPI, focusing on Generator Tool docs. Of the six writers from 2022 GSoD, she distinguished herself by **crafting the most documentation** and **automating the copying of docs from the `/generator` repo to the `/website` repo** via [asyncapi/generator#872](https://github.com/asyncapi/generator/pull/872).

```mermaid
graph LR
    A(("AsyncAPI Docs' Hall of Fame"))
    B["Inaugural Member"]
    C["Florence Njeri"]
    D["Role: Core Maintainer of Generator Tool Docs"]
    E["Key Contribution: Creation & Maintenance of Generator Tool Docs"]
    F["Active Member of AsyncAPI TSC"]

>     G["Impecable Responsiveness as Core Docs Maintainer"]

Reviewer 2: ```suggestion
    G["Impeccable Responsiveness as Core Docs Maintainer"]
```

Author: nice catch, thanks!!
